moses/vae/model.py

```diff
@@ -123,10 +123,6 @@ def forward_decoder(self, x, z):
 
         x = nn.utils.rnn.pad_sequence(x, batch_first=True,
                                       padding_value=self.pad)
-        x_emb = self.x_emb(x)
```

Comment:
why delete this line? Without it, you're passing in the raw token indices to your RNN

Reply:
You're totally right, should be fixed now. Thanks for checking!